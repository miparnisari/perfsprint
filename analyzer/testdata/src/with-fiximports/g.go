package a

import ( // want "fiximports: Fix imports"
	"fmt"
)

func fmtg() {
	// fmt still used after removal, but new strconv
	fmt.Printf("Hello, World!", fmt.Sprintf("%d", 123)) // want "integer-format: fmt.Sprintf can be replaced with faster strconv.Itoa"
}
