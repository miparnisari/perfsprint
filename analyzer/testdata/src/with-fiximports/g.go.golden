package a

import ( // want "fiximports: Fix imports"
	"fmt"
	"strconv"
)

func fmtg() {
	// fmt still used after removal, but new strconv
	fmt.Printf("Hello, World!", strconv.Itoa(123)) // want "integer-format: fmt.Sprintf can be replaced with faster strconv.Itoa"
}
